.github/workflows/e2e.yaml

```diff
@@ -40,7 +40,39 @@ jobs:
           aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Why are there not equivalent constructs for the vsphere settings? Using different mechanisms to configure the two different infrastructure providers makes it seem as if something is missing.

Reply:
https://github.com/NVIDIA/k8s-device-plugin/pull/871/files#r1706671562